src/coinbase/coinbase.ts

```diff
@@ -95,6 +98,15 @@ export class Coinbase {
       basePath: basePath,
     });
     const axiosInstance = globalAxios.create();
+    axiosRetry(axiosInstance, {
+      retries: maxNetworkRetries,
+      retryCondition: (error: AxiosError) => {
+        return (
+          error.config?.method?.toUpperCase() === "GET" &&
+          (error.response?.status || 0) in [500, 502, 503, 504]
```

Comment:
One concern I have with retrying `500` is that we are not setting idempotency keys on all of our APIs